lib/bibdata_rs/src/theses/dataspace/document.rs

```diff
@@ -0,0 +1,174 @@
+// This module is responsible for representing Dataspace Document's metadata
+
+use serde::de::Deserializer;
+use serde::{Deserialize, Serialize};
+
+mod builder;
+mod normalize;
+
+#[derive(Clone, Debug, Serialize)]
```

Comment:
Do we need Clone here?

Reply:
Yes!

Reply:
I pushed up a commit that doesn't need to implement `Clone`